MODULE.bazel

```diff
@@ -0,0 +1,10 @@
+module(
+    name = "googletest",
+    compatibility_level = 1,
+    version = "0.0.0",
```

Comment:
In order not to have to maintain a version number multiple times (or at all), it seems to be a somewhat often used practice to have `0.0.0` here and just increment the version number in the bazel central registry.

Reply:
The version attribute is optional - maybe leave it out, instead of using 0.0.0